Marlin/src/module/temperature.cpp

```diff
@@ -309,19 +309,19 @@ PGMSTR(str_t_heating_failed, STR_T_HEATING_FAILED);
 #endif
 
 #if EITHER(AUTO_POWER_E_FANS, HAS_FANCHECK)
-  uint8_t Temperature::autofan_speed[HOTENDS]; // = { 0 }
+  uint8_t Temperature::autofan_speed[HOTENDS] = {FAN_OFF_PWM};
```

Comment:
C++: All items to init the array must be included. Items after the last one are assumed to be 0.